apollo-compiler/src/main/kotlin/com/apollographql/apollo3/compiler/codegen/CodegenLayout.kt

```diff
@@ -20,7 +20,7 @@ import com.apollographql.apollo3.compiler.singularize
  * Inputs should always be GraphQL identifiers and outputs are valid Kotlin/Java identifiers.
  */
 internal abstract class CodegenLayout(
-    ir: Ir,
+    val ir: Ir,
```

Comment:
I exposed this at one point to get access to the list of enums and custom scalars. I believe I ended up not needing to do this so I will revert.